cmd/http-stats.go

```diff
@@ -35,11 +37,15 @@ type ConnStats struct {
 // Increase total input bytes
 func (s *ConnStats) incInputBytes(n int) {
 	s.totalInputBytes.Add(uint64(n))
+	// Update prometheus metric
+	networkSentBytes.Add(float64(n))
```

Comment:
I think it makes sense if we update networkSentBytes only in updateGeneralMetrics(). In the former function, we can access to total input bytes information instead of updating that information twice, do you see any inconvenient for this solution @sinhaashish ?

Reply:
@vadmeste  : I will check on this and get back.

Reply:
@sinhaashish, in `cmd/globals.go`, there is globalConnStats, you can call globalConnStats.getTotalOutputBytes() to have an update of how much bytes are sent to the network. Will this work ?

Reply:
@vadmeste  : Hi have done the required changes .Please review.